src/lime/_internal/backend/native/NativeAudioSource.hx

```diff
@@ -386,6 +386,11 @@ class NativeAudioSource
 
 	public function setCurrentTime(value:Int):Int
 	{
+		if (value == getCurrentTime())
```

Comment:
This is bad, This makes the current time reset every time it stops, Which is bad.

Reply:
You should remove it imo lol

Reply:
I see the issue. `stop()` and `play()` both call `setCurrentTime(0)`, and the latter gets ignored because the time is already 0.

This would normally be fine. You should skip a setter if it's already in the right place. But `setCurrentTime()` has a second purpose: stopping and starting the audio. Seems like the wrong place for that, but maybe it's necessary? Wish I could think of some better way to do all this...

Ok, I'll comment it out.

Reply:
It seemed fine, but for some reason, it gives a check of played when it gives that, so I guess that you don't neeed it very much.